Assets/Scripts/Game/Items/LootTables.cs

```diff
@@ -93,6 +93,52 @@ public static LootChanceMatrix GetMatrix(string key)
             return DefaultLootTables[0];
         }
 
+        public static bool GenerateLoot(DaggerfallLoot loot, int locationIndex)
+        {
+            string[] lootTableKeys = {
+            "K", // Crypt
+            "N", // Orc Stronghold
+            "N", // Human Stronghold
+            "N", // Prison
+            "K", // Desecrated Temple
+            "M", // Mine
+            "M", // Natural Cave
+            "Q", // Coven
+            "K", // Vampire Haunt
+            "U", // Laboratory
+            "D", // Harpy Nest
+            "N", // Ruined Castle
+            "L", // Spider Nest
+            "F", // Giant Stronghold
+            "S", // Dragon's Den
+            "N", // Barbarian Stronghold
+            "M", // Volcanic Caves
+            "L", // Scorpion Nest
+            "N", // Cemetery
+            };
+
+            // Get loot table key
+            if (locationIndex < lootTableKeys.Length)
+            {
+                DaggerfallLoot.GenerateItems(lootTableKeys[locationIndex], loot.Items);
+
+                // Randomly add map
+                char key = lootTableKeys[locationIndex][0];
+                int alphabetIndex = key - 64;
+
+                if (alphabetIndex >= 10 && alphabetIndex <= 15) // between keys J and 0
```

Comment:
This 0 should be an O, can you fix this typo please? :)

Reply:
The typo is from me, when I added this stuff in before.

Reply:
I thought you meant the 0 in the index, changing that to 'O' broke any inerior with loot. whoops, reverted that crap. Done the correct change to the comment.  (couldn't see until I scrolled right)